include/ibamr/INSVCStaggeredHierarchyIntegrator.h

```diff
@@ -282,6 +282,11 @@ class INSVCStaggeredHierarchyIntegrator : public INSHierarchyIntegrator
     virtual void
     registerBrinkmanPenalizationStrategy(SAMRAI::tbox::Pointer<IBAMR::BrinkmanPenalizationStrategy> brinkman_force);
 
+    /*!
+     * \brief Register function to compute the source term of the div U equation.
```

Comment:
Is this an optional function?

Reply:
(I mean the function to compute the source term, not the class member. 😁 )

Reply:
Yes. It is optional. By default \div u = 0 is used in the code. Only when `d_Div_U_F_fcn` is registered, we solve \div u \ne 0.

Reply:
@boyceg `registerFluidSourceFunction()` is defined in `INSHierarchyIntegrator`. We don't know what it is suppose to be doing. It is defined in `INSStaggeredHierarchyIntegrator` that cannot handle variable density, so it cannot be $\nabla \cdot \bf{u} = -1/\rho \frac{D \rho}{D t}$. We thought it purpose is different than the low Mach model, hence we defined a new function. 

If  `registerFluidSourceFunction` is supposed to be low Mach model, we can use this function instead of defining `registerDivergenceVelocitySourceFunction` in INSVC.

Reply:
`registerFluidSourceFunction` gives a way to set $\nabla \cdot {\bf u} = Q` for the supplied `Q` `CartGridFunction`. It seems like that is the same thing that `registerDivergenceVelocitySourceFunction` is doing?

Reply:
Yes. We will then use the existing function.

Alternatively, we can rename registerfluidSourceFunction to registerDivergenceVelocityFunction to make it more clear?

Reply:
That would be fine, although we should deprecate the old function and have it call the new one. Suggest `registerVelocityDivergenceFunction`.